File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
    List<Integer> sumList = new ArrayList<>();
        
        for(int i=0; i<numbers.length; i++){
            for(int j=i+1; j<numbers.length; j++){
                int sum = numbers[i]+numbers[j];
                if(!sumList.contains(sum)){
                    sumList.add(sum);
                }
            }
        }
        
        Collections.sort(sumList);
        
        int[] answer = new int[sumList.size()];
        
        for(int i=0; i<answer.length; i++){
            answer[i]=sumList.get(i);
        }
        return answer;
    }
}           

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
    var answer = [];
    
    for(var i=0; i<numbers.length; i++){
        for(var j=i+1; j<numbers.length; j++){
            var sum = numbers[i] + numbers[j];
            if(answer.indexOf(sum) == -1){
                answer.push(sum);
            }
        }
    }
    
    answer = answer.sort((a,b) => a-b);
    
    
    
    return answer;
}